Dockerfile

```
@@ -0,0 +1,41 @@
+FROM node:18-alpine AS base
+
+WORKDIR /app
+
+# ----------------------------------------
+# Stage 1: Prepare package.json files
+FROM base AS installer
+
+# COPY the whole project to the container
+COPY . . // NOSONAR
```

Hadolint (2.12.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)